spec/spec_helper.rb

```diff
@@ -113,7 +117,7 @@ def image_path(path, options = {})
 end
 
 VCR.configure do |c|
-  c.cassette_library_dir = Rails.root.join('spec/vcr_cassettes')
+  c.cassette_library_dir = File.join(external_tower_gem_dir, 'spec/vcr_cassettes')
```

Comment:
another reason why the specs shouldnt rely on an external gem.
maybe you can wrap the `it_behaves_like` block with a `VCR` block? Not sure if thats possible and supported though, something like

```ruby
VCR.with_cassette_library_dir('..') do
  it_behaves_like '...'
end
```